docs/asl/alerting-philosophy.md

```diff
@@ -0,0 +1,141 @@
+It is recommended for all alerts to adhere to the follow guidelines:
+
+1. [Keep conditions simple](#keep-it-simple).
+2. [Alerts should be actionable](#actionable-alerts).
+3. [Check for measured failure on critical paths](#check-for-measured-failure) rather than a lack of success.
+4. [Alerts should not have special cases](#avoid-special-cases) for routine maintenance.
+5. [Consider how the alert check can fail](#think-about-failure).
+
+## Keep It Simple
+
+When an alert triggers, it should be easy to understand why. Similarly, if an alert doesn't fire, then it should be easy to check and see what happened. The more complicated an alert condition becomes, the harder it is to understand and debug.
```

Comment:
We generally try to wrap lines in the markdown at 100 characters to make diffs easier to compare and read.